libs/ui-lib/src/components/mull-button/mull-button.scss

```
@@ -0,0 +1,12 @@
+@import '../../lib/styles/variables.scss';
+
+.mull-button {
+  color: white;
+  font-weight: bold;
+  padding: 1rem 3rem;
+  background: $primary-color;
+  border-radius: 9px;
+
+  border: none;
```

Button doesn't have a border by default anyway in our app.
```suggestion
.mull-button {
  color: white;
  font-weight: bold;
  padding: 1rem 3rem;
  background: $primary-color;
  border-radius: 9px;
```

---

I think this was left over due to jimmy and jon putting a border for some reason on the original button I moved out of create event.

I probably set it to none just in case, even though its not needed, much like the rest of the dead css in the rest of this pr haha